acceptance-tests/tests/build.gradle

```diff
@@ -12,8 +12,8 @@
  */
 
 plugins {
-  id 'org.web3j' version '4.11.3'
-  id 'org.web3j.solidity' version '0.4.1'
+  id 'org.web3j' version '4.12.2'
```

Comment:
```suggestion
  id 'org.web3j' version '4.12.3'
```

Reply:
yes the gradle checkLicense task fails - I've asked the q in Besu discord https://discord.com/channels/905194001349627914/905205502940696607/1339482205839298612